.github/workflows/go.yml

```diff
@@ -38,3 +38,16 @@ jobs:
 
     - name: Build
       run: make
```

Comment:
is this redundant since you've added another build below?

Reply:
You're right, I have indirectly added another build because the docker build does a build itself. There's actually some redundancy already because `make` and `make tests` both depend on `make manifests` and `make docker-build` depends on `make test`. I planned to modify how `make tests` is executed in the future, but I think I'm going to look to clean this up some now.